platform/src/main/resources/db/migration/V6__sep24_field_updates.sql

```diff
@@ -0,0 +1,59 @@
+ALTER TABLE sep24_transaction ADD amount_expected VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD kyc_verified VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD message VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD more_info_url VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD refund_memo VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD refund_memo_type VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD refunded BOOLEAN;
+
+ALTER TABLE sep24_transaction ADD refunds JSON;
+
+ALTER TABLE sep24_transaction ADD request_asset_code VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD request_asset_issuer VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD required_info_message VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD status_eta VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD stellar_transactions JSON;
+
+ALTER TABLE sep24_transaction ADD transfer_received_at TIMESTAMP WITHOUT TIME ZONE;
+
+ALTER TABLE sep24_transaction ADD updated_at TIMESTAMP WITHOUT TIME ZONE;
+
+ALTER TABLE sep24_refund_payment DROP CONSTRAINT fk_sep24_refund_payment_on_transaction;
+
+DROP TABLE sep24_refund_payment CASCADE;
+
+ALTER TABLE sep24_transaction DROP COLUMN sep_transaction_id;
+
+ALTER TABLE sep24_transaction DROP COLUMN asset_code;
+
+ALTER TABLE sep24_transaction DROP COLUMN asset_issuer;
+
+ALTER TABLE sep24_transaction DROP COLUMN muxed_account;
+
+ALTER TABLE sep24_transaction DROP COLUMN sep10account_memo;
+
+ALTER TABLE sep24_transaction DROP COLUMN completed_at;
+
+ALTER TABLE sep24_transaction DROP COLUMN started_at;
+
+ALTER TABLE sep24_transaction ADD completed_at TIMESTAMP WITHOUT TIME ZONE;
+
+ALTER TABLE sep24_transaction ALTER COLUMN  id SET NOT NULL;
+
+ALTER TABLE sep31_transaction DROP COLUMN refunds;
+
+ALTER TABLE sep31_transaction ADD refunds JSON;
```

Comment:
I think we should copy data from this column to a new (historic) table for bitso
```suggestion
CREATE TABLE sep31_historic (
   refunds VARCHAR(255)
);

INSERT INTO sep31_historic (refunds)
SELECT refunds
FROM sep31_transaction;

ALTER TABLE sep31_transaction DROP COLUMN refunds;

ALTER TABLE sep31_transaction ADD refunds JSON;
```

Reply:
It's just to keep data that was in refunds table. With the SQL we currently have, we just drop this column entirely.

Reply:
Hey @Ifropc This is lovely and awesome!!! 💯 

💡 Is it possible to check if this SQL script work?

Reply:
Should work, but would be nice to manually test it with some data in postgres

Reply:
Hey @Ifropc I have merged your suggestions to the PR. Can you check if this takes care of the Jira ticket? https://stellarorg.atlassian.net/browse/ANCHOR-171

Reply:
Yes, it's an alternative way to resolve this ticket. The original suggestion was to migrate data, with this new table there is no need for migration. Instead, historic records are stored in the separate table